File: packages/libsql/ios/Plugin/Classes/Results/ConnectResult.swift
import Foundation
import Capacitor

@objc public class ConnectResult: NSObject, Result {
    let connectionId: String

    init(connectionId: String) {
        self.connectionId = connectionId
    }

    @objc public func toJSObject() -> AnyObject {
        var result = JSObject()
        result["connectionId"] = connectionId
        return result as AnyObject
    }
}

File: packages/libsql/android/src/main/java/io/capawesome/capacitorjs/plugins/libsql/classes/results/ConnectResult.java
package io.capawesome.capacitorjs.plugins.libsql.classes.results;

import androidx.annotation.NonNull;
import com.getcapacitor.JSObject;
import io.capawesome.capacitorjs.plugins.libsql.interfaces.Result;

public class ConnectResult implements Result {

    @NonNull
    private final String connectionId;

    public ConnectResult(@NonNull String connectionId) {
        this.connectionId = connectionId;
    }

    @Override
    @NonNull
    public JSObject toJSObject() {
        JSObject result = new JSObject();
        result.put("connectionId", connectionId);
        return result;
    }
}
